### ISSUE TITLE
A number of the links in the readme are broken

### ISSUE THREAD

**Issue by RallyTronics** [MethanePowered/MethaneAsteroids#13]
such as https://github.com/MethanePowered/MethaneKit/tree/master/Modules/Graphics/Extensions/Include/Methane/Graphics/SkyBox.h

**Comment by egorodet** [comment]
@RallyTronics  Yeah, that's true. Some files have been moved in the main repository, while sample was not updated. By the way, you may submit pull-request to fix links.
